lib/arm/cpu_features.c

```diff
@@ -116,4 +118,34 @@ void setup_cpu_features(void)
 	_cpu_features = features | ARM_CPU_FEATURES_KNOWN;
 }
 
+#elif defined(__aarch64__) /* not __linux__ */
+
+volatile u32 _cpu_features = 0;
```

Comment:
Move this to just below `#if ARM_CPU_FEATURES_ENABLED`, so it's one place for the Linux and non-Linux cases.